tests-behat/grid.feature

```diff
@@ -36,3 +36,17 @@ Feature: Grid
     When I click using selector "xpath(//div[@id='grid']//tr[2]//a)"
     Then No toast should be displayed
     Then PATCH MINK the url should match "~_unit-test/grid-rowclick.php#test~"
+
+  Scenario: master checkbox
+    Given I am on "_unit-test/grid-rowclick.php"
+    When I press menu button "Show Selection" using selector ".ui.menu.atk-grid-menu"
+    Then Toast display should contain text "Selected:"
+    When I click using selector "xpath(//div[@id='grid']//tr[1]//div.ui.child.checkbox)"
+    Then I press menu button "Show Selection" using selector ".ui.menu.atk-grid-menu"
+    Then Toast display should contain text "Selected: 1"
+    When I click using selector "xpath(//div[@id='grid']//tr//div.ui.master.checkbox)"
+    Then I press menu button "Show Selection" using selector ".ui.menu.atk-grid-menu"
+    Then Toast display should contain text "Selected: 1,2,3,4,5"
+    When I click using selector "xpath(//div[@id='grid']//tr//div.ui.master.checkbox)"
+    Then I press menu button "Show Selection" using selector ".ui.menu.atk-grid-menu"
+    Then Toast display should contain text "Selected:"
```

Comment:
fix EOL and test something that will text empty selection reliably

Reply:
OK, I now understand that "should contain text" do exactly what is say. It would be true for any string that contains the text and more characters and thus does not test for empty. I add "#" as EOL such as empty appears like "Selected: #".

Reply:
![image](https://user-images.githubusercontent.com/2228672/200143904-e3047d48-0495-42bf-ba6f-87183b409057.png)

by EOL I meant: each text file should have exactly one LF at the end

Reply:
Right! Done.